Tests/InstantSearchCoreTests/Unit/Searcher/SingleIndexSearcherTests.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  SingleIndexSearcherTests.swift
+//  
+//
+//  Created by Vladislav Fitc on 03/08/2020.
+//
+
+import Foundation
+import XCTest
+@testable import InstantSearchCore
+
+class SingleIndexSearcherTests: XCTestCase {
```

Comment:
Do you think we can test `shouldTriggerSearchForQuery` here by checking that no search has been made?